Shared/TimeConstants.swift

```diff
@@ -114,6 +114,11 @@ extension Date {
 extension Date {
     public static var yesterday: Date { return Date().dayBefore }
     public static var tomorrow: Date { return Date().dayAfter }
+    
+    public static var second: Double = 1
```

Comment:
Since Date deals with TimeIntervals, can we change this to be TimeInterval too, instead of Double?